test/fixtures/rules/std_rule_cases/L003.yml

```
@@ -1100,3 +1097,54 @@ test_tsql_bubble_up_newline_after_fix:
   configs:
     core:
       dialect: tsql
+
+test_tsql_cross_apply_indentation:
+  # Test for behavior in issue #3672
+  pass_str: |
+    SELECT
+      table1.col,
+      table2.col
+    FROM table1
+    CROSS APPLY (
+      VALUES ((1), (2))
+    ) AS table2(col)
+    INNER JOIN table3
+      ON table1.col = table3.col;
+  configs:
+    core:
+      dialect: tsql
+
+test_tsql_cross_join_indentation:
+  # Test for behavior in issue #3672
+  pass_str: |
+    SELECT
+      table1.col,
+      table2.col
+    FROM table1
+    CROSS JOIN table2
+    INNER JOIN table3
+      ON table1.col = table3.col;
+  configs:
+    core:
+      dialect: tsql
+
+test_tsql_nested_join:
+  # Test for behavior prior to issue #3672
+  fail_str: |
+    SELECT
+      table1.col,
+      table2.col
+    FROM table1
+    INNER JOIN table2
+    INNER JOIN table3
+      ON table1.col = table2.col AND table1.col = table3.col;
+  fix_str: SELECT
+    table1.col,
+    table2.col
+    FROM table1
+    INNER JOIN table2
+    INNER JOIN table3
+    ON table1.col = table2.col AND table1.col = table3.col;
```

This test case looks wrong. I think you mean this:

```suggestion
  fail_str: |
    SELECT
        table1.col,
        table2.col
    FROM table1
    INNER JOIN table2
    INNER JOIN table3
        ON table1.col = table2.col AND table1.col = table3.col;
  fix_str: |
    SELECT
        table1.col,
        table2.col
    FROM table1
    INNER JOIN table2
        INNER JOIN table3
            ON table1.col = table2.col AND table1.col = table3.col;
```